ja/lessons/ecto/basics.md

```diff
@@ -190,18 +190,18 @@ end
 
 この `Example.Person` は、それが `people` テーブルと関連するものであることをEctoに伝え、stringの `name` とintegerでデフォルトが0の `age` という2つのカラムを持っていることがわかります。
```

Comment:
`Example.Person` 🤔 
maybe`Friends.Person`

I'll try to send pullrequest `en/lessons/ecto/basics.md` next time 👍

Reply:
I sent pullrequest 💪 
https://github.com/elixirschool/elixirschool/pull/1923